sonic-xcvrd/tests/test_xcvrd.py

```diff
@@ -23,13 +22,17 @@
 swsscommon.ProducerStateTable = MagicMock()
 swsscommon.SubscriberStateTable = MagicMock()
 swsscommon.SonicDBConfig = MagicMock()
+#swsscommon.Select = MagicMock()
 
 test_path = os.path.dirname(os.path.abspath(__file__))
 modules_path = os.path.dirname(test_path)
 scripts_path = os.path.join(modules_path, "xcvrd")
 sys.path.insert(0, modules_path)
 
 os.environ["XCVRD_UNIT_TESTING"] = "1"
+from xcvrd.xcvrd import *
+from xcvrd.xcvrd_utilities.sfp_status_helper import *
+from xcvrd.xcvrd_utilities.port_mapping import *
```

Comment:
these imports are already there at line 1 to 3

Reply:
fixed, can you please approve again